### PR TITLE
accept disclaimer in .env, fix ms health check

### PR DESCRIPTION
**Pull request by fredo**
This PR does two things:

- fix ms health check due to typo
- disclaimer must be accepted in .env file in order for the services to run

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -38,6 +38,18 @@ KEYSTORE_FILE=
 PASSWORD=
 ETH_RPC=
 
+# The pathfinding service disclaimer needs to be accepted by setting ACCEPT_SERVICES_DISCLAIMER as written below.
+# Otherwise the pathfinding service won't start running.
```
Comment (ulope) [inline .env.template]:
```suggestion
# Otherwise the pathfinding service won't start.
```

File: .env.template [hunk 2]
```diff
@@ -38,6 +38,18 @@ KEYSTORE_FILE=
...
+# Please make sure to read the disclaimer and only uncomment once you have read and accepted the disclaimer.
+# Please read: https://github.com/raiden-network/raiden-services/blob/f4bcb9c289e093754204fe18684e4b57558ea29b/src/pathfinding_service/constants.py#L34
+#ACCEPT_PFS_DISCLAIMER="--accept-disclaimer"
+
+# The monitoring service disclaimer needs to be accepted by setting ACCEPT_SERVICES_DISCLAIMER as written below.
+# Otherwise the monitoring service won't start running.
```
Comment (ulope) [inline .env.template]:
```suggestion
# Otherwise the monitoring service won't start.
```

File: docker-compose.yml [hunk 3]
```diff
@@ -41,6 +41,7 @@ services:
       - PFS_KEYSTORE_FILE=/keystore/${KEYSTORE_FILE}
       - PFS_PASSWORD=${PASSWORD}
       - PFS_ETH_RPC=${ETH_RPC}
+      - PFS_ACCEPT_DISCLAIMER=${ACCEPT_PFS_DISCLAIMER}
```
Comment (ulope) [inline docker-compose.yml]: Seems to be duplicate (in cli and here)

File: docker-compose.yml [hunk 4]
```diff
@@ -67,8 +68,9 @@ services:
       - MS_KEYSTORE_FILE=/keystore/${KEYSTORE_FILE}
       - MS_PASSWORD=${PASSWORD}
       - MS_ETH_RPC=${ETH_RPC}
+      - MS_ACCEPT_DISCLAIMER=${ACCEPT_MS_DISCLAIMER}
```
Comment (ulope) [inline docker-compose.yml]: Same as above

File: docker-compose.yml (hunk 3 above)
Comment (palango) [inline docker-compose.yml]: Is this necessary with the command above?

File: docker-compose.yml (hunk 4 above)
Comment (palango) [inline docker-compose.yml]: same as above

File: docker-compose.yml [hunk 5]
```diff
@@ -67,8 +68,9 @@ services:
...
     healthcheck:
-      test: 'python3 -c "import sqlite3, sys; from time import sleep; conn=sqlite3.connect(\"/state/ms-state.db\"); r = lambda: conn.execute(\"select latest_commited_block from blockchain ORDER BY latest_commited_block DESC LIMIT 1;\").fetchone()[0]; one = r(); sleep(25); two = r(); print(two); sys.exit(0 if two > one else 1)"'
+      test: 'python3 -c "import sqlite3, sys; from time import sleep; conn=sqlite3.connect(\"/state/ms-state.db\"); r = lambda: conn.execute(\"select latest_committed_block from blockchain ORDER BY latest_committed_block DESC LIMIT 1;\").fetchone()[0]; one = r(); sleep(25); two = r(); print(two); sys.exit(0 if two > one else 1)"'
```
Comment (palango) [inline docker-compose.yml]: Nice find!

File: docker-compose.yml (hunk 3 above)
Comment (palango) [reply docker-compose.yml]: maybe it's nice to remove it from the command, and only use it here. And then define it as `ACCEPT_PFS_DISCLAIMER=TRUE`

Comment (fredo) [reply docker-compose.yml]: yes I think it is, since this command will run in a different environment

**Pull request merged**